docs/source/faq.md

```diff
@@ -10,12 +10,12 @@ and relatively easy way to construct widely used HPC protocols: MPI tag matching
 RMA operations, rendezvous protocols, stream, fragmentation, remote atomic operations, etc.
 
 #### What is UCP, UCT, UCS?
-* **UCT** is a transport layer that abstracts the differences across various hardware architectures and provides a low-level API that enables the implementation of communication protocols. The primary goal of the layer is to provide direct and efficient access to hardware network resources with minimal software overhead. For this purpose UCT relies on low-level drivers provided by vendors such as InfiniBand Verbs, Cray's uGNI, libfabrics, etc. In addition, the layer provides constructs for communication context management (thread-based and ap- plication level), and allocation and management of device- specific memories including those found in accelerators. In terms of communication APIs, UCT defines interfaces for immediate (short), buffered copy-and-send (bcopy), and zero- copy (zcopy) communication operations. The short operations are optimized for small messages that can be posted and completed in place. The bcopy operations are optimized for medium size messages that are typically sent through a so- called bouncing-buffer. Finally, the zcopy operations expose zero-copy memory-to-memory communication semantics.
+* **UCT** is a transport layer that abstracts the differences across various hardware architectures and provides a low-level API that enables the implementation of communication protocols. The primary goal of the layer is to provide direct and efficient access to hardware network resources with minimal software overhead. For this purpose, UCT relies on low-level drivers provided by vendors such as InfiniBand Verbs, Cray's uGNI, libfabrics, etc. In addition, the layer provides constructs for communication context management (thread-based and application level), and allocation and management of device specific memories including those found in accelerators. In terms of communication APIs, UCT defines interfaces for immediate (short), buffered copy-and-send (bcopy), and zero-copy (zcopy) communication operations. The short operations are optimized for small messages that can be posted and completed in place. The bcopy operations are optimized for medium size messages that are typically sent through a so-called bouncing-buffer. Finally, the zcopy operations expose zero-copy memory-to-memory communication semantics.
```

Comment:
remove "etc."

across -> between

Reply:
@tonycurtis
1. Do you mean the ```etc``` in ```UCT relies on low-level drivers provided by vendors such as InfiniBand Verbs, Cray's uGNI, libfabrics, etc``` ?
2. Do we need to change ```across```  -> ```between``` or ```across``` -> ```among``` or just keep it there?

Reply:
So it's actually "libfabric" I think.  The "etc" is not needed: when you list examples following "such as" or "e.g." then you just put the examples you want.  The "etc" is only possible when both author and readership are expected to know the other possibilities.  You also typically say "differences between" things instead of "across" or "among".

Reply:
@tonycurtis Done: remove ```etc```